File: core/src/main/java/com/neqsoft/reso/utils/view/HeaderViewHolder.java
package com.neqsoft.reso.utils.view;

import android.view.View;
import android.widget.TextView;

import com.neqsoft.reso.R;

import androidx.annotation.NonNull;
import androidx.appcompat.widget.AppCompatImageView;
import androidx.recyclerview.widget.RecyclerView;

public class HeaderViewHolder extends RecyclerView.ViewHolder {

  private TextView titleTv;
  private AppCompatImageView arrowIv;

  public HeaderViewHolder(@NonNull View itemView) {
    super(itemView);
    titleTv = itemView.findViewById(R.id.titleTv);
    arrowIv = itemView.findViewById(R.id.arrowIv);
  }

  public void setTitle(@NonNull String title) {
    titleTv.setText(title);
  }
}

File: core/src/main/kotlin/com/neqsoft/reso/utils/view/HeaderViewHolder.kt
package com.neqsoft.reso.utils.view

import android.view.View
import android.widget.TextView
import androidx.appcompat.widget.AppCompatImageView
import androidx.recyclerview.widget.RecyclerView
import com.neqsoft.reso.R

class HeaderViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    val titleTv: TextView = itemView.findViewById(R.id.titleTv)
    val arrowIv: AppCompatImageView = itemView.findViewById(R.id.arrowIv)
}